torchdynamo/convert_frame.py

```diff
@@ -294,44 +309,44 @@ def debug_print(prefix):
                 except exc.SkipFrame:
                     return None
```

Comment:
Can we add a log here as well? I spent a lot of time once to figure where my frame went.

Reply:
And similarly a log just before - on the RestartAnalysis